### PR TITLE
Change references to "central server"

### LINKED ISSUE #330: Change references to "Central Server"
The term "central server" isn't really descriptive of what it is any more, and has always been a bit confusing in discussions of servers. There is persistent (if vague and unsubstantiated) evidence that newcomers to Jamulus assume that running a server in some form is required for any use. Distancing the terminology ("directory" vs "central server") may help with that, and helps to make the word "server" mean what is currently sometimes called a "normal server" in conversations about the two types (particularly when this involves non-technical people being instructed to connect to a "custom central server").

So we'll replace it with the term "Directory Server" (as the formal name), and in most cases just "Directory" so as to disambiguate between "normal" servers. Eg:

"Configuring your server as a Central Server ..." should be "Configuring your server as a Directory" 

"You need to enter a custom central server address" should be "You need to enter a custom directory address" 

Also need to change this in the diagram (and update the logo there)

### PR DESCRIPTION
**Pull request by gilgongo**
# Does this need translation?

- [x] Yes 
- [ ] No 

# Changes

As per https://github.com/jamulussoftware/jamuluswebsite/issues/330 This PR does a simple word replacement. No phraseology has changed.

Note that the option `--centralserver` still exists to confuse everyone :-)

### CODE REVIEW

**Review by ann0see — APPROVED** [review]
Did you check if all links work correctly? 

Sorry for being late. I might miss some PRs/have missed some...

**Comment by gilgongo** [comment]
I changed the file name from en-Central-Servers.md to en-Directory-Servers.md and the links to that accordingly (in all files in /wiki/en) so I hope it's OK..

**Comment by ann0see** [comment]
Ok. You could run a link checker locally and check if anything is broken. Also maybe we'd need to set up a redirect from Central server to directory server?

**Comment by gilgongo** [comment]
I'll check locally. Do redirects have to be client-side on GHPages? Would rather not have to have maintain a blank page.

**Comment by ann0see** [comment]
Yes. Unfortunately. See e.g. https://github.com/jamulussoftware/jamuluswebsite/blob/release/wiki/en/en-Multiple-Audio-Interfaces.md

**Comment by pljones** [comment]
> Note that the option --centralserver still exists to confuse everyone :-)

Easy :).

Duplicate `--centralserver` but for a new `--directoryserver` option (i.e. support both but have them do exactly the same thing - don't even worry about having both specified).  Then replace `--centralserver` in the `--help` output with `--directoryserver`.

Mmm, like...
https://github.com/jamulussoftware/jamulus/pull/1407
that.

File: wiki/en/en-Choosing-a-Server-Type.md [hunk 1]
```diff
@@ -46,17 +46,18 @@ Jamulus --nogui --server
 
 See also [Command Line Options](Command-Line-Options) for other parameters you can set.
 
-### 3. Central
-Configuring your server as a Central Server need only be done under special circumstances (e.g. for online events or musical associations). Most people can ignore this type.
+### 3. Directory
+Configuring your server as a Directory Server need only be done under special circumstances (e.g. for online events or musical associations). Most people can ignore this type.
 
-To view servers listed by a custom central server, musicians must enter the address in their client’s "Custom Central Server" settings field.  They will then see a Connection Setup list which is generated by that central server.
+To view servers listed by a custom directory server, musicians must enter the address in their client’s "Custom Directory Server" settings field.  They will then see a Connection Setup list which is generated by that directory server.
```
Comment (hoffie) [inline wiki/en/en-Choosing-a-Server-Type.md:52]: Along with changing the docs, we have to update multiple places in application labels as well, right?
```
$ grep -ci central src/res/translation/translation_de_DE.ts 
28
```

(Adding this here as "Custom Central Server" is being changed here)

**Review by hoffie — COMMENTED** [review]
Should we have some "Note: Directory servers used to be called Central servers in the past" somewhere to avoid confusion by people who come from older discussions/issues/external documentation?

File: wiki/en/en-Choosing-a-Server-Type.md (hunk 1 above)
Comment (pljones) [reply wiki/en/en-Choosing-a-Server-Type.md:52]: Grows a bit, doesn't it?  But the English should be all there now... if I did it right.  (Ugh... no, didn't do the ts files...)

Comment (pljones) [reply wiki/en/en-Choosing-a-Server-Type.md:52]: OK, so now the ts files are there, too.

**Comment by gilgongo** [comment]
OK so all links checked and nothing was broken, but I've added back the old en-Central-Servers.md file with a redirect in the unlikely event anyone is linking to it. 

I guess we need to wait until the codebase changes have been made now.

**Comment by ann0see** [comment]
@gilgongo can we merge this (after the conflicts have been resolved?) I think both PRs are waiting on eachother

**Pull request merged**